extension-config/popup.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
I went with whatever for the markup and tota11y icon. If you would like this tweaked, let me know.